styles/styles.css

```diff
@@ -1,3 +1,96 @@
+/* здесь начинается Колина часть css */
+
+body {
+    box-sizing: border-box;
+    margin: 0;
+    font-family: 'Work Sans', 'sans-serif';
+}
+
+.hiring {
+    display: flex;
+    flex-direction: column;
+    width: 320px;
+    height: 866px;
+    background-color: #fff   ;
+}
+
+.title_hiring {
+    margin-top: 50px;
+    margin-bottom: 0px;
+    font-weight: 700;
+    text-align: center;
+    color: #2B292D;
+    font-size: 40px;
+}
+
+.description_hiring {
+    margin: 18px auto 30px;
+    top: 113px;
+    width: 290px;
+    height: 136px;
+    font-weight: 400;
+    font-size: 24px;
+    text-align: center;
+}
+
+.more_about_company {
+    margin: 0px auto 39px;
+    padding: 0;
+    width: 290px;
+    height: 48px;
+    border-style: none;
+    background: linear-gradient(180deg, rgba(255, 255, 255, 0.05) 0%, rgba(255, 255, 255, 1e-05) 100%), #3040C4;
+    background-blend-mode: overlay, normal;
+    box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1), 0px 4px 6px rgba(0, 0, 0, 0.05), inset 0px 1px 0px rgba(255, 255, 255, 0.1);
+    border-radius: 2px;
+    color: #fff;
+    font-size: 18px;
+    font-weight: 700;
+    cursor: pointer;
+}
+
+.vacancies {
+    width: 290px;
+    height: 140px;
+    border-radius: 8px;
+    margin: 0 auto 15px;
+    border: 1px solid rgba(43, 41, 45, 0.2);
+    cursor: pointer;
+}
+.vacancies_title {
+    width: 199px;
+    height: 69px;
+    margin: 19px 71px 0px 20px;
+    text-align: left;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 28px;
+    color: #2B292D;
+}
+.vacancies_location {
+    width: 199px;
+    height: 26px;
+    font-weight: 400;
+    color: #2B292D;
+    opacity: 60%;
+    font-size: 16px;
+    line-height: 22px;
+    margin: 7px 71px 19px 20px;
+    text-align: left;
+}
+.vacancies_action-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
```

Comment:
Do you need an `absolute` position here?

Reply:
rebuild this in the next commit. please, verify